lib/compilation.sh

```diff
@@ -485,6 +498,12 @@ CUSTOM_KERNEL_CONFIG
 		fi
 	fi
 
+	# Uniquely define variables before building source and binary packages:
+	# kernel package name extension, local version, debian package version
+	NAME_EXTENSION="${NAME_EXTENSION:-${BRANCH}-$LINUXFAMILY}"
+	LOCALVERSION="-$LINUXFAMILY$ADDITIONAL_LOCALVERSION"
+	KDEB_PKGVERSION=${version}-${VENDOR}.${REVISION%-*}
```

Comment:
Perhaps UPPERCASE ?
KDEB_PKGVERSION=${version}-${VENDOR,}.${REVISION%-*}

linux-dtb-edge-meson64_5.19.0-Armbian.22.08.0_arm64.deb -> linux-dtb-edge-meson64_5.19.0-ARMBIAN.22.08.0_arm64.deb

Reply:
This question seems very simple, but maybe we will try to maintain the style of the basic distributions?
And by the way, it has to do with who is doing the assembly. If a user collects locally, can he use the "Armbian" brand?

Reply:
> This question seems very simple

Yes its cosmetic issue.

> And by the way, it has to do with who is doing the assembly. If a user collects locally, can he use the "Armbian" brand?

Technically its Armbian, but unofficial. I would not be changing this. Perhaps some PGP signed kernel or similar?